manifest/openshift4/coreos/cluster-network-22-nsx-node-agent.yaml

```diff
@@ -135,4 +138,6 @@ spec:
         name: dbus
       - hostPath: {path: /etc/sysconfig/network-scripts}
         name: nscripts
+      - hostPath: {path: /var/log/nsx-ujo}
```

Comment:
Looks like it is already in line 129

Reply:
the host-var-log-ujo in line 129 is mounted as /var/log/openvswitch, you can see line 86

Reply:
removed the dup.